ghcide/src/Development/IDE/Spans/AtPoint.hs

```diff
@@ -308,6 +311,24 @@ atPoint IdeOptions{} (HAR _ hf _ _ (kind :: HieKind hietype)) (DKMap dm km) env
             UnhelpfulLoc {} | isInternalName name || isSystemName name -> Nothing
             _ -> Just $ "*Defined " <> printOutputable (pprNameDefnLoc name) <> "*"
 
+        prettyInstances :: (Either ModuleName Name, IdentifierDetails hietype) -> IO (Maybe T.Text)
+        prettyInstances (Right n, _) =
```

Comment:
In cases like this I'd usually just make the function take what it needs (just a `Name`) and get the caller to worry about massaging the list into the right form.

Reply:
Sounds better, I switched it up